ModularTegustation/tegu_items/refinery/crates/corporation.dm

```diff
@@ -107,29 +110,32 @@
 	icon_state = "crate_rcorp"
 	veryrarechance = 5
```

Comment:
This needs it's crate count increased to 3.5 and given a bunch more R-Corp equipment.